README.md

```diff
@@ -52,6 +52,12 @@ problems. Community support is also available in the
 [LND Slack](https://lightning.engineering/slack.html)
 .
 
+### Regtest Loop server
+
+To get started with local development against a stripped down dummy Loop server
+running in a local `regtest` Bitcoin network, take a look at the
+[`regtest` server environment example documentation](./regtest/README.md).
+
```

Comment:
btw I've moved this text into my README PR as well. Not sure how to deal with conflicts but just a heads up there. Added you as a reviewer to that too

Reply:
Ah, you're using the GitHub web UI? I'm not sure whether/how it can handle conflicts. Let me know if you run into problems, I can rebase the branch for you.